src/test/test-util.c

```diff
@@ -13,11 +13,14 @@
 #include "raw-clone.h"
 #include "rm-rf.h"
 #include "string-util.h"
+#include "tests.h"
 #include "util.h"
 
 static void test_align_power2(void) {
         unsigned long i, p2;
 
+        log_info("/* %s */", __func__);
```

Comment:
maybe turn this into a macro? log_test_func()?

Reply:
It wouldn't save much. In various places there's additional information being printed (for example args for the test function when its called multiple times).